### PR TITLE
install redis-exporter

### PR DESCRIPTION
**Pull request by nonsense**
TODO:
- [x] install `redis-exporter` helm chart, that deploys a service which exposes /metrics endpoint
- [x] scrape that /metrics endpoint and get measurements into prometheus
- [x] visualise measurements in grafana
- [x] automate installation of exporter, scraping config
- [ ] automate import of grafana dashboard

### CODE REVIEW

File: infra/k8s/redis-exporter-values.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#       key: redis-password
+#       name: redis-config-0.0.2
+service:
+  type: ClusterIP
+  port: 9121
+  annotations: {}
+  labels: {}
+resources: {}
+redisAddress: redis://redis-master:6379
+annotations:
```
Comment (nonsense) [inline infra/k8s/redis-exporter-values.yaml]: Based on what I read, adding those `annotations` should tell `prometheus` to scrape that endpoint, but this doesn't seem to be working, so I've done something wrong.

Comment (nonsense) [reply infra/k8s/redis-exporter-values.yaml]: Note to self - read the full values.yaml, no matter how long they are (facepalm)

**Comment by Robmat05** [comment]
@coryschwartz Good points. So let’s either amend the ticket by removing the dashboard requirement item, and creating a new ticket in the backlog for a future milestone. Or simply delete it if we don’t think we’re going to address this issue anytime soon

**Comment by coryschwartz** [comment]
I want to cancel *most* of this, since a separate node exporter is not required. I have enabled the node exporter in the existing redis values with just this line, and the prometheus operator picks this up  fine. https://github.com/ipfs/testground/pull/640/files#diff-719513b66178c36b1cd1c887da1b08a1R469

However, I do want to keep the actual dashboard.

Since @nonsense isn't here right now I'm going to go ahead and close this, but I'll create a PR with the same dashboard

**Pull request closed without merging**